### DISCUSSION TITLE
Is there a MAUI CT gallery/showcase app?

### DISCUSSION THREAD

**Discussion by stamminator** [CommunityToolkit/Maui#2080] [General]
Similar to the Windows Community Toolkit Gallery app or the unofficial [WinUI Community Gallery app](https://github.com/WinUICommunity/WinUICommunity), is there an app which demos the various controls and features available through this project?

**Comment by TheCodeTraveler** [comment]
Of course! We have a [sample app](https://github.com/CommunityToolkit/Maui/tree/main/samples) in this repo that contains samples for every feature in the Toolkit 💯

**Reply by stamminator** [reply]
Right on! I'll check it out. I'm also now seeing the reference to it in the v9.0.2 release.

Are there plans to include an installer of the sample app in future release assets? Being able to just run the app and get a feel from the user's perspective is something I like to do to gauge the state of a project before getting into the source code. Believe it or not, the Microsoft Store listing is actually how I discovered the official WinUI 3 Gallery app.

**Reply by TheCodeTraveler** [reply]
You can install the app onto any iOS / Android / macOS / Windows device using the latest version of Visual Studio to deploy `CommunityToolkit.Maui.Sample.sln`. 

We don't have plans to publish the app to the App Stores.

**Reply by stamminator** [reply]
Gotcha. Putting installers in the release assets would be sufficient and much easier than publishing to app stores, but I know that's still extra work to set up.

**Reply by VladislavAntonyuk** [reply]
We may look at it after migrating from Azure DevOps to Github

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
